syfosm-common-models/src/main/kotlin/no/nav/syfo/model/sykmelding/model/AktivitetIkkeMuligDTO.kt

```diff
@@ -0,0 +1,6 @@
+package no.nav.syfo.model.sykmelding.model
+
+data class AktivitetIkkeMuligDTO(
+    val medisinskArsak: MedisinskArsakDTO?,
```

Comment:
Hvis denne brukes for EnkelSykmelding så skal vel ikke medisinsk vurdering være med..?

Reply:
Åja, det der er ikke medisinks vurdering

Reply:
ah sant :D